badge-maker/lib/badge-renderers.js

```diff
@@ -576,78 +580,111 @@ function forTheBadge({
   links,
   logo,
   logoWidth,
-  logoPadding,
   color = '#4c1',
   labelColor,
 }) {
-  // For the Badge is styled in all caps. Convert to caps here so widths can
-  // be measured using the correct characters.
```

Comment:
I think this comment is still relevant (and helpful), but a few lines further down where we call `.toUpperCase()`